vars/publishToMaven.groovy

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/** Library to sign and deploy opensearch artifacts to a sonatype staging repository.
+ @param Map args = [:] args A map of the following parameters
+ @param args.signingArtifactsPath <required> - Local Path to yml or artifact file. This could be path to Dir that contains yml file or full path to yml file.
```

Comment:
Didn't quite understand this description (quite confusing). Can you be more precise?